src/org/thoughtcrime/securesms/ConversationAdapter.java

```diff
@@ -103,6 +99,8 @@
   private final @NonNull  Calendar          calendar;
   private final @NonNull  MessageDigest     digest;
 
+  private @Nullable String searchTerm;
```

Comment:
Maybe align it with the top variables if possible (to keep things consistent)

Reply:
it is not a final variable so I decided it would look better as its own subset of variables.